### DISCUSSION TITLE
is there any way to not make vae encoder crop the image?

### DISCUSSION THREAD

**Discussion by sabi3293043** [Comfy-Org/ComfyUI#431] [Q&A]
the vaeEncoder is cropping the image no matter what i do this is not ideal for img2img. if it was rescaling i would be fine with it but its straight up cropping.

**Comment by sabi3293043** [comment]
i figured it out. i have attached a solution

i could not attach the json so heres an image with the comfy ui config
i just padded the image before making it a latent leaving some extra black area
then cropped it back to its origenal size

![ComfyUI_00123_](https://user-images.githubusercontent.com/30566201/230487756-4bed6867-834b-4e7f-b3f4-8d7df8e7462e.png)

**Comment by comfyanonymous** [comment]
Yes right now images need to be in sizes that are multiples of 64 pixels which is why it gets cropped.

**Reply by WASasquatch** [reply]
> Yes right now images need to be in sizes that are multiples of 64 pixels which is why it gets cropped.

You should just enforce a multiple instead of altering the image. That's not good practice. Even if it warps the image. But most the time it will be in aspect ratio, just as a multiple of 8.
